dependencies.gradle

```diff
@@ -28,6 +28,7 @@ ext.versions = [
 
         junit                  : '4.13.2',
         turbine                : '0.7.0',
+        stately                : '1.2.0'
```

Comment:
Is this unused?

Reply:
What the...?!? Sorry about that, must have snuck in there when rebasing and rearranging my git history.